core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -120,17 +123,17 @@ object ZkSecurityMigrator extends Logging {
   }
 }
 
-class ZkSecurityMigrator(zkUtils: ZkUtils) extends Logging {
+class ZkSecurityMigrator(zkClient: KafkaZkClient) extends Logging {
   private val futures = new Queue[Future[String]]
 
   private def setAcl(path: String, setPromise: Promise[String]) = {
     info("Setting ACL for path %s".format(path))
-    zkUtils.zkConnection.getZookeeper.setACL(path, zkUtils.defaultAcls(path), -1, SetACLCallback, setPromise)
+    zkClient.currentZooKeeper.setACL(path, zkClient.defaultAcls(path).asJava, -1, SetACLCallback, setPromise)
   }
 
   private def getChildren(path: String, childrenPromise: Promise[String]) = {
     info("Getting children to set ACLs for path %s".format(path))
-    zkUtils.zkConnection.getZookeeper.getChildren(path, false, GetChildrenCallback, childrenPromise)
+    zkClient.currentZooKeeper.getChildren(path, false, GetChildrenCallback, childrenPromise)
```

Comment:
`currentZooKeeper` is meant to be used for testing only.

Reply:
@ijuma Thanks for the review. maybe we should allow  for this tool alone.

Other options are:
1. implementing similar behavior in KafkaZKCleint methods
2. using ZooKeeper instance directly in ZkSecurityMigrator

What do you suggest?

Reply:
Personally, I'd replace this async style with the pipelined client. This is unnecessarily complex.

Reply:
Understood. If it's ok, I will do the change in another PR.

Reply:
@ijuma  Thanks for the suggestion. Yes, this will make more clear to the developers. added ZkSecurityMigratorUtils class to to access zookeeper instance,